src/main/java/software/bernie/geckolib3/renderers/geo/ExtendedGeoEntityRenderer.java

```diff
@@ -337,7 +426,7 @@ protected void renderArmorPart(MatrixStack stack, ModelRenderer sourceLimb, int
 
 	protected void renderBlock(MatrixStack matrixStack, IRenderTypeBuffer rtb, int packedLightIn,
 			BlockState iBlockState) {
-		// TODO: Re-implement, doesn't do anything yet
+		// TODO: Re-implement, doesn't do anyhting yet
```

Comment:
![geckolib](https://media4.giphy.com/media/1AgUVT7uaYOoHa5FVh/giphy.gif)
Please change this back XD

Reply:
Uhm ye, can do that, honestly i don't know how it got there lol, i didn't touch that line